Cargo.toml

```diff
@@ -14,6 +14,8 @@ rustc-hex = "2.0"
 secp256k1 = "0.12"
 serde = { version = "1.0", features = ["derive"]}
 parity-crypto = "0.3"
+lazy_static = "1.0"
 
 [dev-dependencies]
 serde_json = "1.0"
+rand = "0.6"
```

Comment:
Let's make the tests deterministic for now  and avoid rand, we can add quick-check later.